app/src/pages/inside/common/infoLine/infoLine.stories.jsx

```diff
@@ -0,0 +1,35 @@
+import { storiesOf } from '@storybook/react';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+// eslint-disable-next-line import/extensions, import/no-unresolved
+import { WithState } from 'storybook-decorators/withState';
+import { InfoLine } from './infoLine';
+import { state, mockData } from './data';
+import README from './README.md';
+
+const withToolTip = (getStory) => {
```

Comment:
I think it can be moved to common decorators `.storybook/decorators`

Reply:
Done